### ISSUE TITLE
Be able to use up/down cursor keys to nudge spacing field values

### ISSUE THREAD

**Issue by dgwyer** [tomusborne/generateblocks#1135]
I love being able to use the cursor keys in browser dev tools to nudge numerical values up/down. Maybe we can consider this too for spacing input boxes (padding, margin, border width etc.)?

**Comment by tomusborne** [comment]
This would be cool.

One potential challenge here is that our `UnitControl` can access values like `calc()` and `var()`, so we would need logic to only allow this if the value is numeric.

**Comment by dgwyer** [comment]
I've noticed that the GitHub auto-close issue feature doesn't seem to be working. e.g. I marked this issue to be closed via the PR [here](https://github.com/tomusborne/generateblocks/pull/1147) but it didn't seem to work.

Not sure exactly why, but I think it could be to do with it only auto-closing when merging a PR into the default branch (i.e. `master` in this case). It's pretty annoying to have to manually close associated issues!
